tests/image/test_list.py

```diff
@@ -32,12 +32,15 @@ def test_cli_no_scrapinghub_config():
 
 
 @pytest.mark.usefixtures('project_dir')
+@pytest.mark.parametrize('is_binary_logs', [True, False])
 @mock.patch('shub.image.utils.get_docker_client')
 @mock.patch('requests.get')
-def test_cli(requests_get_mock, get_docker_client_mock):
+def test_cli(requests_get_mock, get_docker_client_mock, is_binary_logs):
     """Case when shub-image-info succeeded."""
     requests_get_mock.return_value = _get_settings_mock()
     mocked_logs = json.dumps({'project_type': 'scrapy', 'spiders': ['abc', 'def']})
+    if is_binary_logs:
+        mocked_logs = mocked_logs.encode('utf-8')
```

Comment:
I luckily didn't have to use Python 2 for quite a while and have lost touch a bit with the whole str/unicode mess, so I may be mistaken here.

In Py2, won't `json.dumps` already return a utf8-encoded `str`, and we're double-encoding it here, testing one `utf-8` encoded `str` and one double-encoded `str` but never `unicode`?

Reply:
@jdemaeyer No, you're right - that's my bad! Updated the PR with a small helper for `str` conversions, could you please take a look if it covers the issue or not?